### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

**Comment by joohoi** [comment]
Thanks for the PR! The tool looks really good too. I didn't have the time to go through all of the options, but I'll definately (intended typo, as it's one of the words I tend to write wrong all the time) will be using this in the future. I wonder if there's a suggested way to automate all this in Travis job for example, but I'll create an feature request issue in your repo regarding this.

**Comment by jsoref** [comment]
@joohoi https://github.com/checkstyle/checkstyle/blob/master/.ci/test-spelling-unknown-words.sh is what I did for the checkstyle project. I've been meaning to offer it to other projects.

**Comment by joohoi** [comment]
I'll check it out later on, thanks!

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Using acme-dns is a three-step process (provided you already have the self-hoste
 
 The method returns a new unique subdomain and credentials needed to update your record.
 Fulldomain is where you can point your own `_acme-challenge` subdomain CNAME record to.
-With the credentials, you can update the TXT response in the service to match the challenge token, later referred as \_\_\_validation\_token\_recieved\_from\_the\_ca\_\_\_, given out by the Certificate Authority.
+With the credentials, you can update the TXT response in the service to match the challenge token, later referred as \_\_\_validation\_token\_received\_from\_the\_ca\_\_\_, given out by the Certificate Authority.
```
Comment (joohoi) [inline README.md:43]: Fun fact: https://github.com/joohoi/acme-dns/pull/66#issuecomment-379536205

**Review by joohoi — APPROVED** [review]
LGTM! Thanks!

File: README.md (hunk 1 above)
Comment (jsoref) [reply README.md:43]: You aren't alone: https://github.com/ossec/ossec-docs/pull/252/commits/1b447f032aceab026b57c4761ed115e4f6bb6b56 (and following)

**Pull request merged**